test/over_react/util/css_value_util_test.dart

```diff
@@ -27,12 +27,12 @@ main() {
           double value = 34.0;
           var units = ['em', 'ex', 'rem', 'vh', 'vw', 'vmin', 'vmax', '%', 'px', 'cm', 'mm', 'in', 'pt', 'pc', 'ch'];
           for (String unit in units) {
-            expect(CssValue.parse('$value$unit').number, equals(value));
+            expect(CssValue.parse('$value$unit')!.number, equals(value));
```

Comment:
It looks like by default it fails to parse it returns null, and yeah since we currently support returning null from a custom `onError`, it needs to be nullable.